wire-ios-sync-engine/Source/UserSession/UserSession.swift

```diff
@@ -218,7 +218,16 @@ public protocol UserSession: AnyObject {
 
     func fetchAllClients()
 
+<<<<<<< HEAD
     // MARK: - Use Cases
+=======
+    func createTeamOneOnOne(
+        with user: UserType,
+        completion: @escaping (Swift.Result<ZMConversation, CreateTeamOneOnOneConversationError>) -> Void
+    )
+
+    // MARK: Use Cases
+>>>>>>> 4a6000814a (feat: update 1:1 creation flow and handle errors WPB-6938 (#1321))
```

Comment:
conflict not resolved here

Reply:
https://github.com/wireapp/wire-ios/pull/1362/commits/b9cd928a8a243d7457bec36b0f00d6612909aea2